package.json

```diff
@@ -47,5 +47,8 @@
     "prettier": "^2.0.5",
     "typescript": "^3.8.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@babel/preset-env": "^7.10.3",
+    "tsdx": "^0.13.2"
+  }
```

Comment:
Are you sure these are meant to be added to the root package json?

Reply:
I was getting many yarn workspace errors when I was trying to add packages specifically to the ui-docs repo, this fixed it. The errors were:

`error An unexpected error occurred: "expected workspace package to exist for \"PACKAGE NAME HERE\"".`